File: Web_nghe_nhac_online/setup-structure.js
const fs = require('fs');
const path = require('path');

const basePath = path.join(__dirname, 'src', 'main', 'resources', 'static');

const directories = [
    path.join(basePath, 'pages'),
    path.join(basePath, 'pages', 'admin'),
    path.join(basePath, 'css'),
    path.join(basePath, 'js')
];

directories.forEach(dir => {
    if (!fs.existsSync(dir)) {
        fs.mkdirSync(dir, { recursive: true });
        console.log(`✓ Created: ${dir}`);
    } else {
        console.log(`✓ Exists: ${dir}`);
    }
});

console.log('\n✓ Directory structure created successfully!');

File: Web_nghe_nhac_online/SetupStructure.java
import java.io.File;

public class SetupStructure {
    public static void main(String[] args) {
        String basePath = "src/main/resources/static";
        
        String[] directories = {
            basePath + "/pages",
            basePath + "/pages/admin",
            basePath + "/css",
            basePath + "/js"
        };
        
        for (String dir : directories) {
            File directory = new File(dir);
            if (!directory.exists()) {
                if (directory.mkdirs()) {
                    System.out.println("✓ Created: " + dir);
                } else {
                    System.out.println("✗ Failed to create: " + dir);
                }
            } else {
                System.out.println("✓ Exists: " + dir);
            }
        }
        
        System.out.println("\n✓ Directory structure setup complete!");
    }
}
